scripts/revng

```diff
@@ -4,16 +4,19 @@ import argparse
 import os
 import subprocess
 import sys
+from pathlib import Path
 
-from binascii import hexlify
-from ctypes.util import find_library
 try:
     from shutil import which
 except ImportError:
     from backports.shutil_which import which
 
-from revng_translate import register_translate, run_translate
-from revng_support import build_command_with_loads, set_verbose, get_command, run
+python_libs_path = str(Path(__file__).parent.parent / "lib" / "python")
+sys.path.insert(0, python_libs_path)
+os.environ["PYTHONPATH"] = f"{python_libs_path}:" + os.environ["PYTHONPATH"]
```

Comment:
Please double check the semantics of `PYTHONPATH`, because having an empty entry in `PYTHONPATH` might have undesired side effects.
For `PATH`, `PATH=/lol:` is different from `PATH=/lol`, IIRC it adds to the search path `.`, which is a serious (unintended) change.

Also, let's add it only if the directory exists.

Reply:
Ping?